extensions/css-language-features/server/src/utils/documentContext.ts

```diff
@@ -32,6 +40,16 @@ export function getDocumentContext(documentUri: string, workspaceFolders: Worksp
 					}
 				}
 			}
+			// Following [css-loader](https://github.com/webpack-contrib/css-loader#url)
+			// and [sass-loader's](https://github.com/webpack-contrib/sass-loader#imports)
+			// convention, if an import path starts with ~ then use node module resolution
+			// *unless* it starts with "~/" as this refers to the user's home directory.
+			if (ref[0] === '~' && ref[1] !== '/') {
```

Comment:
I'm not sure how this can be unit tested, I guess I'd need to mock require.resolve, I don't see an equivalent test for
https://github.com/Microsoft/vscode-languageserver-node/blob/master/server/src/resolve.ts